src/app/catalog-count/catalog-count.component.ts

```diff
@@ -1,21 +1,65 @@
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+import { DateTimeHandler } from '../shared/datetime-handler';
 import { CatalogCountService } from './catalog-count.service';
 
 @Component({
-  selector: 'cc-catalog-count',
   templateUrl: './catalog-count.component.html',
   styleUrls: ['./catalog-count.component.scss'],
 })
 export class CatalogCountComponent implements OnInit {
-  catalogCounts: any[] = [];
+  pageTitle = 'Favor de capturar otro registro (Catálogo de Cuentas)';
+  catalogCountForm!: FormGroup;
   errorMessage = '';
+  ccEnums: any[] = [];
 
-  constructor(private ccService: CatalogCountService) {}
+  constructor(
+    private fb: FormBuilder,
+    private router: Router,
+    private ccService: CatalogCountService
+  ) {
+    this.catalogCountForm = this.fb.group({
+      amount: ['', Validators.required],
+      details: ['', Validators.required],
+      registrationDate: [
+        new Date().toISOString().substring(0, 10),
+        Validators.required,
+      ],
+      catalogCountEnum: ['', Validators.required],
+    });
+  }
 
   ngOnInit(): void {
-    this.ccService.getCatalogCounts().subscribe({
-      next: (data) =>
-        (this.catalogCounts = data.catalogCountResponseCollection),
+    this.loadCcEnums();
+    //toISOString -> UTC
+  }
+
+  saveCatalogCount(): void {
+    if (this.catalogCountForm.valid) {
+      const payload = { ...this.catalogCountForm.value };
+      let payloadRegistration = new Date(payload.registrationDate);
+      payload.registrationDate =
+        (payloadRegistration.getTime() +
+          DateTimeHandler.getCurrentTimeZoneMilliseconds() +
+          DateTimeHandler.getDateTimeOffSetMilliseconds()) /
+        1000;
```

Comment:
keep an eye on this!

compare how values are being stored in DB.. (intention UTC)